ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx

```diff
@@ -24,6 +25,12 @@ type ComponentProps = {
       content: StreamResponse | null;
       contributors: string | null;
     };
+    atiAnalytics: {
+      contentId: string;
+      contentType: string;
+      pageIdentifier: string;
+      pageTitle: string;
+    };
```

Comment:
You may be able to use the type defined in https://github.com/bbc/simorgh/blob/latest/src/app/components/ATIAnalytics/types.ts#L67 for this